cpp/src/bindings.cpp

```diff
@@ -625,8 +625,17 @@ PYBIND11_MODULE(_pypowsybl, m) {
     
     m.def("create_flow_decomposition", &pypowsybl::createFlowDecomposition, "Create a security analysis");
 
-    m.def("add_precontingency_monitored_elements_for_flow_decomposition", &pypowsybl::addPrecontingencyMonitoredElementsForFlowDecomposition, "Add elements to be monitored for a flow decomposition",
-          py::arg("flow_decomposition_context"), py::arg("elements_ids"));
+    m.def("add_contingency_for_flow_decomposition", &pypowsybl::addContingencyForFlowDecomposition, "Add a contingency for flow decomposition",
```

Comment:
Is it needed to specified "for flow decomposition" it is in the module with the same name ? what do you think ?

Reply:
ok it is in _pypowsybl so it is needed